### PR TITLE
Fixes GLUE RTE example

### PR DESCRIPTION
**Pull request by ashwinsr01**
- Fixed data interface 
- Fix stale train.py imports
- Fix infer.py checkpoint directory
- Checkpoint to true without which infer wont work

### CODE REVIEW

File: examples/GLUE/src/infer.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 def load_classifier():
     config = CustomArgParser().parse()
-    checkpoint_path = '/home/azureuser/GLUE/checkpoints/model_9.pt'
+    checkpoint_path = 'checkpoints/model_9.pt'
```
Comment (ashwinsr01) [inline examples/GLUE/src/infer.py:6]: @krishansubudhi Modified this to relative directory. I git cloned pymarlin and tried the example from within examples/glue. Let me know if you think its due to the I tried the example.

Comment (ashwinsr01) [reply examples/GLUE/src/infer.py:6]: This definitely due to my approach of cloning entire pymarlin. Still feel relative directory would work for all VM.

Comment (krishansubudhi) [reply examples/GLUE/src/infer.py:6]: necessary. Thanks

**Review by krishansubudhi — APPROVED** [review]
Thanks for making these changes. Looks good.

**Pull request merged**